cmd/kubelet/app/server.go

```diff
@@ -240,7 +240,7 @@ func (s *KubeletServer) Run(_ []string) error {
 
 	// TODO(vmarmol): Do this through container config.
 	if err := util.ApplyOomScoreAdj(0, s.OOMScoreAdj); err != nil {
-		glog.Info(err)
+		glog.V(4).Info(err)
```

Comment:
This is printed once at startup, is there a particular reason to make it such a low V value? Knowing if this triggered helps a lot under OOM conditions.